guides/common/modules/proc_installing-the-discovery-service.adoc

```diff
@@ -35,6 +35,7 @@ ifdef::orcharhino[]
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # {foreman-installer} \
+--enable-foreman-cli-discovery \
```

Comment:
I had a look on various other insllation sections. Like https://github.com/ATIX-AG/foreman-documentation/blob/12bb012474bc04d032ba745a587fdd07af7d491f/guides/common/modules/proc_installing-the-remote-execution-plugin.adoc?plain=1#L18 
Sometimes the cli installation is integrated in the usual install sections (like in salt), sometimes it is separate. Is there a generic rule how to do it? I would prefer a consistent way (and then I would prefer to have it separate)

Reply:
There is currently no consistent way, but the docs assume that users always have the option to interact with Foreman via WebUI, API, CLI, and Ansible. Therefore, it makes sense to install the Hammer CLI plugins by default when installing optional Foreman/Katello plugins.